lib/gli/commands/help_modules/command_help_format.rb

```diff
@@ -13,38 +13,38 @@ def initialize(command,app,sorter,synopsis_formatter_class,wrapper_class=TextWra
         end
 
         def format
-          command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
-          wrapper              = @wrapper_class.new(Terminal.instance.size[0],4)
+          @command_wrapper      = @wrapper_class.new(Terminal.instance.size[0],4 + @command.name.to_s.size + 3)
```

Comment:
The changes here are the most odd to me, because making this ivars definitely makes the code more complex, since it increases the scope of all these variables.  They are only needed in this method but making them ivars makes them global to the class.

I guess that the warnings are from inside the ERB template?  What if we inlined the ERB template to this method?